docs/general/naming.md

```diff
@@ -222,9 +225,37 @@ Examples:
 Metric `http.server.request.duration` uses attributes from the registry such as
 `server.port`, `error.type`.
 
+### System-specific attributes
+
+**Status**: [Development][DocumentStatus]
+
+When an attribute is specific to a particular system (e.g., project, provider, product),
+the system name should be used in the attribute name, following the pattern:
+`{optional domain}.{system}.*.{property}` pattern.
+
+Examples:
+
+- `db.cassandra.consistency_level` - Describes the consistency level property
```

Comment:
Similar to https://github.com/open-telemetry/semantic-conventions/pull/1708/files#r1894523030

what's the benefit of having `db` in front of `cassandra` here? why not `cassandra.consistency_level`?

Reply:
Let's say we have some redis-specific attributes (and redis can be used as a DB and as a messaging system), would we define `db.redis` and `messaging.redis` ? probably not, it should be just `redis`.